acceptance_modelisation/base_acceptance_map_creator.py

```diff
@@ -453,10 +494,25 @@ def _create_model_cos_zenith_binned(self,
             raise
         per_wobble = i_method < 0
 
-        # Determine initial binning value
+        # Initial binning edge
```

Comment:
Maybe it's just me, but I would put the mentions of bins and edges on the plural form.

Reply:
Done